tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -27,29 +27,34 @@ def content_for_client(module_target_sat, module_org, module_lce, module_cv, mod
 
     :return: Activation key, client lifecycle environment(used by setup_capsules())
     """
-    module_target_sat.cli_factory.setup_org_for_a_custom_repo(
-        {
-            'url': settings.repos.RHEL7_OS,
-            'organization-id': module_org.id,
-            'content-view-id': module_cv.id,
-            'lifecycle-environment-id': module_lce.id,
-            'activationkey-id': module_ak.id,
-        }
-    )
+    rhel_ver = settings.content_host.default_rhel_version
+    baseos = eval(f'settings.repos.RHEL{rhel_ver}_OS.baseos')
```

Comment:
How about versions < 8 which don't have baseos and upstream?

Reply:
My thought process was to keep this simple and not support RHEL 7 but I guess a good question would be how long do we need to support rhel 7? Especially in this one scenario that affects only this test.

Reply:
I think it's up to you, I just pointed it out.